src/vs/base/browser/formattedTextRenderer.ts

```diff
@@ -214,6 +219,8 @@ function formatTagType(char: string): FormatType {
 			return FormatType.Action;
 		case ']':
 			return FormatType.ActionClose;
+		case '`':
```

Comment:
When a backtick happens in a string that does not set `supportPreformattedSegments`, what happens? How does the parser handler `FormatType.Invalid`?

Reply:
Parser treats invalid tokens as normal chars and adds them to the text content of the string.